matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/permalinks/ViaParameterFinder.kt

```diff
@@ -43,6 +43,10 @@ internal class ViaParameterFinder @Inject constructor(
                 .joinToString(prefix = "?via=", separator = "&via=") { URLEncoder.encode(it, "utf-8") }
     }
 
+    fun computeViaParams(viaList: List<String>): String {
```

Comment:
All the fun are called `computeViaParams` is a bit weird.
Also this new fun could be called from the fun above, to avoid code dup.

Reply:
reused and renamed to asUrlViaParameters(List<String>): String